src/styles.css

```diff
@@ -0,0 +1,159 @@
+@import url("https://fonts.googleapis.com/css?family=Source+Sans+Pro&display=swap");
+.App {
+  font-family: sans-serif;
+}
+
+.container{
+  margin: auto;
+  width: 600px;
+  height: 400px;
```

Comment:
Pixels are nice, but what if I run it on a phone with less than 600px width? 
I recommend using the `vw` unit for width, and setting `max-width` with pixels for desktop use.

Reply:
max-width: 600px;
  width: 600vw;
  max-height: 400px;
  height: 400vh;